.github/workflows/node.js.yml

```diff
@@ -20,10 +20,8 @@ jobs:
         with:
           node-version: "14.x"
       - run: npx lerna bootstrap
-      - run: npx lerna run lint
-      - run: npx lerna run typescript
+      - run: npx turbo run lint typescript build
```

Comment:
Unrelated to this PR: It strikes me as weird that the workflows are using `npx`/`npm` while the actual project uses `yarn`. We should probably switch it over.